receiver/httpcheckreceiver/README.md

```diff
@@ -37,6 +37,8 @@ Each target has the following properties:
 
 - `endpoint` (required): the URL to be monitored
 - `method` (optional, default: `GET`): The HTTP method used to call the endpoint
+- `body` (optional, default: ""): If set, the receiver will emit metrics based on whether the response body exact matches this string.
```

Comment:
This could carry a different meaning when method is `POST` where the `body` content will be sent as part of the request.